pimd/pim6_cmd.c

```diff
@@ -1569,6 +1569,117 @@ DEFPY (show_ipv6_pim_nexthop_lookup,
 	return CMD_SUCCESS;
 }
 
+DEFPY (show_ipv6_multicast,
+       show_ipv6_multicast_cmd,
+       "show ipv6 multicast [vrf NAME]",
+       SHOW_STR
+       IPV6_STR
+       "Multicast global information\n"
+       VRF_CMD_HELP_STR)
+{
+	struct vrf *v;
+	struct pim_instance *pim;
+
+	v = vrf_lookup_by_name(vrf ? vrf : VRF_DEFAULT_NAME);
```

Comment:
Same problem here as in PR #10988, if the user gives a VRF name that does not exist, `v` will be NULL.